tests/testthat/test-skip.R

```diff
@@ -75,3 +34,97 @@ test_that("autogenerated message is always single line", {
   # ensure the message is not repeated, #1290
   expect_snapshot_output(cat(cnd$message))
 })
+
+# skip helpers ------------------------------------------------------------
+
+test_that("skip_if_not_installed() works as expected", {
+  expect_snapshot_skip(skip_if_not_installed("doesntexist"))
+  expect_snapshot_skip(skip_if_not_installed("testthat", "9999.9999.999"))
+  expect_no_skip(skip_if_not_installed("testthat", "1.0.0"))
+})
+
+test_that("skip_if_not_installed() works as expected", {
+  skip_if_not_installed("curl")
+  skip_on_cran()
+
+  expect_no_skip(skip_if_offline())
+
+  local_mocked_bindings(has_internet = function(host) FALSE)
+  expect_snapshot_skip(skip_if_offline())
+})
+
+test_that("skip_on_cran() works as expected", {
+  withr::local_envvar(NOT_CRAN = "true")
+  expect_no_skip(skip_on_cran())
+
+  withr::local_envvar(NOT_CRAN = "false")
+  expect_snapshot_skip(skip_on_cran())
+})
+
+
+test_that("skip_on_ci() works as expected", {
+  withr::local_envvar(CI = "false")
+  expect_no_skip(skip_on_ci())
+
+  withr::local_envvar(CI = "true")
+  expect_snapshot_skip(skip_on_ci())
+})
+
+test_that("skip_on_covr() works as expected", {
+  withr::local_envvar(R_COVR = "false")
+  expect_no_skip(skip_on_covr())
+
+  withr::local_envvar(R_COVR = "true")
+  expect_snapshot_skip(skip_on_covr())
+})
+
+test_that("skip_on_bioc() works as expected", {
+  expect_no_skip(skip_on_bioc())
+
+  withr::local_envvar(IS_BIOC_BUILD_MACHINE = "true")
+  expect_snapshot_skip(skip_on_bioc())
+})
+
+test_that("superseded CI skips still work", {
+  expect_no_skip(skip_on_travis())
+  expect_no_skip(skip_on_appveyor())
+
+  withr::local_envvar(TRAVIS = "true", APPVEYOR = "True")
```

Comment:
```suggestion
  withr::local_envvar(TRAVIS = "true", APPVEYOR = "true")
```

Just for consistency